heal_swin/run_configs/segmentation/data_efficiency_swin_hp_synwoodscape_large_plus_AD_train_run_config.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env -S python3 -u
+# fmt: off
+#SBATCH -t 7-00:00:00  # noqa: E265
+#SBATCH -o ../../../slurm/slurm-%j.out  # for array jobs, this should be slurm-%A_%a.out # noqa: E265
+# this is needed to prevent black from formatting the above SBATCH comments...
+dummy="dummy"  # noqa: E225
+# fmt: on
+
+import os  # noqa: E402
+import subprocess  # noqa: E402
+from pathlib import Path  # noqa: E402
+
+import argparse
+import math
+
+def get_argparser_for_config_args():
```

Comment:
This goes a bit against our philosophy of only using config files and staying away from argparse as much as possible. Could you refactor this to be used as an array job, similarly to our evaluation config files? Then, the range of fractions and the seed are also documented here. For array jobs, you also have to change the naming of the slurm file in line 4 and the `job_id` should contain the task number in line 43.